packages/backend-core/src/features/features.ts

```diff
@@ -267,9 +267,9 @@ export class FlagSet<V extends Flag<any>, T extends { [key: string]: V }> {
 // All of the machinery in this file is to make sure that flags have their
 // default values set correctly and their types flow through the system.
 export const flags = new FlagSet({
-  DEFAULT_VALUES: Flag.boolean(env.isDev()),
-  AUTOMATION_BRANCHING: Flag.boolean(env.isDev()),
-  SQS: Flag.boolean(true),
+  [FeatureFlag.DEFAULT_VALUES]: Flag.boolean(env.isDev()),
+  [FeatureFlag.AUTOMATION_BRANCHING]: Flag.boolean(env.isDev()),
+  [FeatureFlag.SQS]: Flag.boolean(true),
```

Comment:
Just realised we are missing the `FeatureFlag.BUDIBASE_AI` from here - can we add it so it goes with this PR?

Reply:
Done!